mock-apns/README.md

```diff
@@ -0,0 +1,28 @@
+# APNs Mock Server
+
+## Setup
+Before you run the mock server you need to create a ssl certificate
+```bash
+openssl req -new -x509 -keyout private -out private -nodes
```

Comment:
You are specifying the same file for private key and certificate. You should use two different files (cert.pem and key.pem for example)